examples/with-apollo-auth/lib/withApollo.js

```diff
@@ -51,7 +50,6 @@ export default App => {
             {...appProps}
             Component={Component}
             router={router}
-            apolloState={apolloState}
```

Comment:
Since we render the whole tree including `App` we have to include all of `App`'s props. This includes apolloState, in this case it's empty since there's no initial state. This is read from the `constructor` though, so it should be an object.

Reply:
Feel free to close if that explains the reason for you.

Reply:
Thank for the explanation Tim but this props is not used in anyway by App component (the one extending next/app), I really don't get it.

Reply:
see https://github.com/zeit/next.js/blob/canary/examples/with-apollo-auth/pages/_app.js#L8

Reply:
Ah you're right.